### PR TITLE
[Odd-34] Refactor removeItemFromUi method

### PR DESCRIPTION
**Pull request by pandyama**
Linear - https://linear.app/oddhours/issue/ODD-34/refactor-removeitemfromui-method

Notes
- created 2 functions in Helper as well as moved companion objects
- tests went well

### CODE REVIEW

**Comment by oyiza** [comment]
let's pair for a bit this evening if you have time. there are some things i'd like us to look through together on this PR

**Comment by pandyama** [comment]
> let's pair for a bit this evening if you have time. there are some things i'd like us to look through together on this PR

yaa totally... does my 7:30pm work? i might have to delay it a little depending on dinner prep, but for now 7:30 is good

**Comment by oyiza** [comment]
> yaa totally... does my 7:30pm work? i might have to delay it a little depending on dinner prep, but for now 7:30 is good

no worries bro that works for me!!

**Comment by pandyama** [comment]
@oyiza its good to review now

File: app/src/main/java/com/example/oddhours/ui/shifts/ChildAdapter.kt [hunk 1]
```diff
@@ -134,10 +133,11 @@ class ChildAdapter(private var shiftsList: List<ShiftsModel>, val context: Conte
                         dayOfYear = 0
                         val helper = Helper()
                         dayOfYear = helper.calculateDayOfTheYear(monthOfYear, dayOfMonth, year)
-                        // DEBUG
-                        // Log.d(TAG, "startDate: year: $year, month: ${monthOfYear}, day: $dayOfMonth")
-                        // Log.d(TAG, "startDate after setting via the button: ${startDate.time}")
                     }, year, month, day)
+                    // DEBUG
```
Comment (oyiza) [inline app/src/main/java/com/example/oddhours/ui/shifts/ChildAdapter.kt]: i may have forgotten but i believe these values are different outside of the datePickerDialog so maybe we should have the debug logging back inside the dialog

File: app/src/main/java/com/example/oddhours/ui/shifts/ChildAdapter.kt [hunk 2]
```diff
@@ -134,10 +133,11 @@ class ChildAdapter(private var shiftsList: List<ShiftsModel>, val context: Conte
...
+                    // Log.d(TAG, "startDate: year: $year, month: ${monthOfYear}, day: $dayOfMonth")
+                    // Log.d(TAG, "startDate after setting via the button: ${startDate.time}")
+                    // TODO: there should be some logic here to move around the maxDate and minDate for the datepicker
```
Comment (oyiza) [inline app/src/main/java/com/example/oddhours/ui/shifts/ChildAdapter.kt]: we can also get rid of this todo comment

File: app/src/main/java/com/example/oddhours/ui/shifts/ChildAdapter.kt (hunk 1 above)
Comment (pandyama) [reply app/src/main/java/com/example/oddhours/ui/shifts/ChildAdapter.kt]: @oyiza could you explain what you mean by logging back inside the dialog. like move the log lines after line 135?

Comment (oyiza) [reply app/src/main/java/com/example/oddhours/ui/shifts/ChildAdapter.kt]: yeah exactly

**Comment by oyiza** [comment]
can we look to close this out so we can close out the cycle?

**Comment by pandyama** [comment]
> can we look to close this out so we can close out the cycle?

yup my bad, this slipped away from me

**Comment by pandyama** [comment]
@oyiza its good to review

**Review by oyiza — APPROVED** [review]

**Pull request merged**